sky/provision/instance_setup.py

```diff
@@ -67,6 +70,26 @@
                             'sky.skylet.attempt_skylet;')
 
 
+def _set_usage_run_id_cmd() -> str:
+    """Gets the command to set the usage run id.
+
+    The command saves the current usage run id to the file, so that the skylet
+    can use it to report the heartbeat.
+
+    We use a function instead of a constant so that the usage run id is the
+    latest one when the function is called.
+    """
+    return (f'cat {usage_constants.USAGE_RUN_ID_FILE} || '
+            f'echo "{usage_lib.messages.usage.run_id}" > '
+            f'{usage_constants.USAGE_RUN_ID_FILE}')
```

Comment:
I might be wrong, but looks like the `run_id` here is set only on the remote cluster (in `head_runner.run`) and will be different the `sky launch` run_id which created the cluster? Do we want to use the same run_id throughout?